src/theme/components/subnav.ts

```diff
@@ -1,6 +1,14 @@
 import { createMultiStyleConfigHelpers } from "@chakra-ui/styled-system";
 import { StyleFunctionProps } from "@chakra-ui/system";
 
+/**
+ * NOTE: The SubNav component does implement responsive spacing, but it does not
```

Comment:
Would it make sense to refactor SubNav so that it _does_ use the responsive spacing pattern like other components? Or is it the DOM structure that makes it different?

Reply:
To allow the button focus rings to show properly, padding was necessary on one of the component's internal elements. And that style application differs from other components.